install-swagger-ui.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015, EMC, Inc.
+
+cd ./static
+rm -rf swagger-ui
+curl -L -o swagger-ui.zip https://github.com/zyoung51/swagger-ui/archive/master.zip
```

Comment:
yep - per conversation in Slack channel, seemed most sensible to keep a local fork in RackHD org instead of a personal one if we're going to use it internally